packages/main/src/Button.js

```diff
@@ -378,7 +378,10 @@ class Button extends UI5Element {
 	}
 
 	get isIconOnly() {
-		return !Array.from(this.childNodes).filter(node => node.nodeType !== Node.COMMENT_NODE).length;
+		return !Array.from(this.childNodes).filter(node =>{
```

Comment:
I think ESLINT will require a space after the `=>` operator here

Reply:
Yes, you are right.